src/5.0_to_5.1/__test_resources__/uiDashboardsFolder.ts

```diff
@@ -0,0 +1,617 @@
+// Contains:
+// 81a - Dashboard with no calculated measures.
+// 8c1 - Empty dashboard.
```

Comment:
I had written this comment as notes to help me remember what the content was of each dashboard id I had added to the test resources when I was writing tests. Is it useful to leave this comment here, or should I remove it ?

Reply:
I think it's nice to have it 👍